.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "depends/googletest"]
 	path = depends/googletest
 	url = ../../google/googletest.git
+[submodule "depends/imgui"]
+	path = depends/imgui
+	url = https://github.com/20k/dfhack-imgui.git
```

Comment:
be sure to use a relative path so the submodule uses the same transport mechanism as the parent fork.

also, before we merge this, we'll want to fork imgui into the DFHack org like most of the dependencies in this file